ui/design-system-core/src/components/Image/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropsWithChildren } from 'react';
+import { apply, tw } from '@twind/core';
+
+export type ImageProps = {
+  assetName?: string;
+  publicImgPath?: string;
+  assetExtension?: string;
+  customStyle?: string;
+  dataTestId?: string;
+};
+
+const Image: React.FC<
+  PropsWithChildren<
+    ImageProps &
+      React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>
+  >
+> = props => {
+  const {
+    src,
```

Comment:
you can simplify this component, no need for assetName, assetExtension and publicImagePath props, you can just pass those to the src directly as a query string

Reply:
okay, let me refactor to use the default HTML props